src/main.rs

```diff
@@ -19,27 +39,54 @@ fn model(app: &App) -> Model {
     let mut app_handler = AppHandler::new();
     app_handler.init();
 
-    let mut sprite_list = SpriteList::new();
-    sprite_list.add_sprite(app, "assets/images/pacman.jpeg","pacman1", 50.0, 50.0);
-    sprite_list.add_sprite_dim(app, "assets/images/pacman.jpeg","pacman2", 100.0, 100.0, 50.0, 50.0);
+    let sprite_list = SpriteList::new();
+    let audio = Arc::new(Audio::new());
 
+    let thread_pool = ThreadPoolBuilder::new().num_threads(4).build().unwrap();
 
-    Model { app_handler, sprite_list }  
+    Model {
```

Comment:
is the model being used as a representation of the frame?

Reply:
No, the model in Nannou represents the management of the global state of the app, it manages the views, updates and events.